scripts/test_stable_diffusion.py

```diff
@@ -0,0 +1,26 @@
+import torch
+from diffusers import AutoPipelineForText2Image
+from transformers import AutoTokenizer
+
+# Set up the pipeline
+model_id = "stabilityai/stable-diffusion-xl-base-1.0"
+pipeline = AutoPipelineForText2Image.from_pretrained(model_id, torch_dtype=torch.float16, variant="fp16", use_safetensors=True)
+
+# Move the pipeline to GPU if available
+device = "cuda" if torch.cuda.is_available() else "cpu"
```

Comment:
nit: add case for "mps", perhaps we could move device selection to somewhere shared in backend?

Reply:
added
this script might be unnecessary anyway